docs/_config.yml

```diff
@@ -1,7 +1,5 @@
-# Site settings
 title: instantsearch.js
 description: instantsearch.js is a library of widgets designed to help you build high performance instant search experiences with Algolia
-baseurl: "/instantsearch.js" # the subpath of your site, e.g. /blog/
```

Comment:
We have to keep the baseurl if we're going to push on github pages. It has to match the name of the repo (the part after `algolia.github.io`) and be used as a prefix in every link in our templates.